tests/integration/src/test/scala/mesosphere/marathon/integration/MesosAppIntegrationTest.scala

```diff
@@ -191,7 +191,7 @@ class MesosAppIntegrationTest extends AkkaIntegrationTest with EmbeddedMarathonT
       val deleteResult = marathon.deletePod(pod.id)
 
       Then("The pod is deleted")
-      deleteResult should be(Deleted)
+      deleteResult should be(Accepted)
```

Comment:
did we change the response code for pod deletion?

Reply:
Nope. This was what we were doing before:

```
val Accepted = new RestResultMatcher(202)
val Deleted = new RestResultMatcher(202)
```

I replaced `Deleted` with `Accepted`, lmk if this is not ideal.